docs/multi-stage-query/reference.md

```diff
@@ -192,12 +192,13 @@ The following ISO 8601 periods are supported for `TIME_FLOOR` and the string con
 - PT1H
 - PT6H
 - P1D
-- P1W
+- P1W*
 - P1M
 - P3M
 - P1Y
 
-For more information about partitioning, see [Partitioning](concepts.md#partitioning-by-time).
+For more information about partitioning, see [Partitioning](concepts.md#partitioning-by-time). <br /><br />
+*Avoid using `P1W` due to alignment complications with months and years.
```

Comment:
```suggestion
*Avoid  partitioning by week, `P1W`, because weeks don't align neatly with months and years, making it difficult to partition by coarser granularities later.
```